cloudSupport/src/main/scala/cromwell/cloudsupport/aws/auth/AwsAuthMode.scala

```diff
@@ -71,18 +71,17 @@ sealed trait AwsAuthMode {
     * All traits in this file are sealed, all classes final, meaning things
     * like Mockito or other java/scala overrides cannot work.
     */
-   private[auth] var credentialValidation: ((AwsCredentials, String) => Unit) =
-     (credentials: AwsCredentials, region: String) => {
-       STSClient
-          .builder
-          .region(Region.of(region))
-          .credentialsProvider(StaticCredentialsProvider.create(credentials))
-          .build
-          .getCallerIdentity(GetCallerIdentityRequest.builder.build)
+   private[auth] var credentialValidation: ((AwsCredentials, Option[String]) => Unit) =
+     (credentials: AwsCredentials, region: Option[String]) => {
+       val builder = STSClient.builder
+       region.foreach(str => builder.region(Region.of(str)))
```

Comment:
I have to admit it took me a bit to grok that this is conditionally setting the region into the statefully built builder... maybe a comment?